acceptable/__main__.py

```diff
@@ -362,7 +362,9 @@ def render_markdown(metadata, cli_args):
 def lint_cmd(cli_args, stream=sys.stdout):
     metadata = load_metadata(cli_args.metadata)
     import_metadata(cli_args.modules)
-    current, locations = get_metadata().serialize()
+    _metadata = get_metadata()
+    current, locations = _metadata.serialize()
+    openapi = _metadata.openapi_31x()
```

Comment:
Have you considered passing the output of `_metadata.serialize()` to e.g. `acceptable.openapi.dumps()` or similar, so that all openapi stuff is in the new module while leaving the APIMetadata class unchanged? (yes, I'm proposing some renaming btw hehe)

Reply:
I've rearranged it a bit. openapi.dump() now takes _metadata as a parameter. Mayyybe the serialize output is good enough, but I don't want to commit to that just yet ;D